elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/AsyncIT.java

```diff
@@ -72,6 +72,15 @@ public AsyncIT() {
         super(AsyncIntegrationTestApplicationResourceConfig.class, JsonApiEndpoint.class.getPackage().getName());
     }
 
+    /**
+     * Returns an initialized data store.
+     *
+     * @return an initialized data store.
+     */
+    public static DataStore getDataStore() {
```

Comment:
What calls this?

Reply:
It is called from IntegrationTest.java

Reply:
Not sure how I feel about using method hiding here.  I think we need to find another way to wire this up.

Reply:
Created an AsyncIntegrationTest to extend AsyncIT from.